tests/utils/suite_writer/test.py

```diff
@@ -80,13 +80,6 @@ def _write_epilogue(self, code_formatter):
         self._write_event(code_formatter, 'test_end')
         super(Test, self)._write_epilogue(code_formatter)
 
-    def _write_cleanups(self, code_formatter):
```

Comment:
@vmalloc I deleted this method because no one uses it & it uses `self._cleanups` which does not exist.